test_utils/enterprise_report_choices.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Enterprise Django application tests constants.
+"""
+
+LIMITED_REPORT_TYPES = {
+    'data_type': [
+        ['progress', 'progress'],
+        ['progress_v2', 'progress_v2'],
+        ['progress_v3', 'progress_v3'],
```

Comment:
Also if there are multiple versions for a specific value, we should send only the latest version. and the display value should not contain version information.
here we should get `['progress', 'progress_v3'],`